.github/workflows/build-and-test.yml

```diff
@@ -414,10 +418,12 @@ jobs:
         run: |
           $ErrorActionPreference = 'Stop'
           Set-PSDebug -Trace 1
+          $msi_file_path = Resolve-Path .\dist\splunk-otel-collector*.msi
           $msi_file_name = Resolve-Path .\dist\splunk-otel-collector*.msi | Split-Path -leaf
-          if ($msi_file_name -match '(\d+\.)(\d+\.)+(\d*)') { $version = $matches[0] }
+          $msi_file_name -match '(\d+\.)(\d+\.)+(\d*)'
+          $version = $matches[0]
           write-host "Building choco package..."
-          .\internal\buildscripts\packaging\choco\make.ps1 build_choco -Version $version | Tee-Object -file .\dist\build_logs.log
+          .\internal\buildscripts\packaging\choco\make.ps1 build_choco -MSIFile $msi_file_path -Version $version | Tee-Object -file .\dist\build_logs.log
```

Comment:
is this an existing oversight or does the default version logic change something here?

Reply:
Existing oversight. This is less fuzzy.
The code does this:
```
    if ($MSIFile -Eq "") {
        $MSIFile = "$BuildDir\splunk-otel-collector-$Version-amd64.msi"
    }
```
However, I didn't see a whole lot of value of leaving it to chance.